### PR TITLE
fix: macos make failed, add go clean cache(clean.sh)

### PR DESCRIPTION
**Pull request by village-way**
macos make failed, add go clean cache(clean.sh), add verify-all.sh for make verify

#### What type of PR is this?

/kind cleanup
#### Which issue(s) does this PR fix?

Fixes #
#### Special notes for your reviewer:

@duanmengkk @ONE7live 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: hack/clean.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${REPO_ROOT}/hack/util.sh"
+
+function clean() {
+  echo "${REPO_ROOT}"/_tmp "${REPO_ROOT}"/_output
+  rm -rf "${REPO_ROOT}"/_tmp "${REPO_ROOT}"/_output
+  util::go_clean_cache
+}
+clean
```
Comment (duanmengkk) [inline hack/clean.sh:26]: add a new line

File: hack/util.sh [hunk 2]
```diff
@@ -527,4 +532,12 @@ function util::wait_for_crd() {
...
   return 1
+}
+
+function util::go_clean_cache() {
+    set -x
+    # clean go cache avoid macos make error
+    # vendor/github.com/prometheus/client_golang/prometheus/expvar_collector.go:18:2: open /usr/local/go/src/expvar: too many open files in system
+    go clean -cache
+    set +x
 }
```
Comment (duanmengkk) [inline hack/util.sh:545]: add a new line

File: hack/clean.sh (hunk 1 above)
Comment (village-way) [reply hack/clean.sh:26]: fixed

```
function clean() {
  echo "${REPO_ROOT}"/_tmp "${REPO_ROOT}"/_output
  rm -rf "${REPO_ROOT}"/_tmp "${REPO_ROOT}"/_output
  util::go_clean_cache
}

clean
```

File: hack/util.sh (hunk 2 above)
Comment (village-way) [reply hack/util.sh:545]: fixed

```
function util::go_clean_cache() {
    set -x

    # clean go cache avoid macos make error
    # vendor/github.com/prometheus/client_golang/prometheus/expvar_collector.go:18:2: open /usr/local/go/src/expvar: too many open files in system
    go clean -cache

    set +x
}
```

**Pull request merged**